### PR TITLE
Add tile rendering for features

### PR DESCRIPTION
**Pull request by hampelm**
Ready for review.

I still want to add a tile zoom limit (specifically return a blank tile at far zooms)

### CODE REVIEW

**Comment by hampelm** [comment]
Will transfer / respond to questions from #144 later today so we can keep track of them

**Comment by hampelm** [comment]
From #147: 

> It'd ideally be nice to factor out the S3/knox putBuffer error handling into a promise-returning function. We'd probably need to wrap the underlying errors or failure scenarios (non-200 status). Then the catch function will mostly be some logic to decide what to log, which is less fragile than deciding that we have indeed encountered an error and passing control to the appropriate continuation.

I think we ought to skip this for now and log it as an issue for later.

**Comment by hampelm** [comment]
@prashtx This is ready for the next round of review. I'm also going to test on my dev server.

File: lib/postgres.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+var knex = require('knex');
+
+var settings = require('./settings');
+
+var pg = knex({
+  client: 'pg',
+  connection: settings.psqlConnectionString
+});
+
+exports.getClient = function getClient() {
```
Comment (prashtx) [inline lib/postgres.js]: Can we just set `module.exports = pg` instead of using a getter function?

**Comment by prashtx** [comment]
Other than the postgres note, LGTM.

File: lib/postgres.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+var knex = require('knex');
+
+var settings = require('./settings');
+
+exports.pg = knex({
```
Comment (hampelm) [inline lib/postgres.js]: @prashtx switched to returning knex obj directly

**Pull request merged**